### ISSUE TITLE
Modify Readme

### ISSUE THREAD

**Issue by jigar-sable** [KISHOREMUTHU/ETE2---Sparky#1]
- Enhance readme file arrange the mockups/screenshots of the application to make it view better.
- Images are too big make it in a tabular way i.e. 3 images per column.

 @KISHOREMUTHU **assign me this issue under LGM-SOC'21**

**Comment by KISHOREMUTHU** [comment]
Yes sure Jigar Sable . So , if you want to edit README , then download that file , edit that and send that to me .

**Comment by jigar-sable** [comment]
Ok I'll work on it and create PR soon..
If possible assign me for this issue.

**Comment by jigar-sable** [comment]
@KISHOREMUTHU review my PR.

**Comment by jayant766** [comment]
Kindly assign this issue ,
So , I can work on it as soon as possible !

**Comment by Ayan-thecodeking** [comment]
hey @KISHOREMUTHU i want to update readme 
please assign me for this ..
if issue is open 
thanks :)
